.project-keeper.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  - type: maven
+    path: pom.xml
+    modules:
+      - jar_artifact
+      - integration_tests
```

Comment:
pom did not contain integration_tests. Is this still correct?

Reply:
Removed :)